azure-pipelines/jobs/ci-linux.yml

```diff
@@ -0,0 +1,82 @@
+parameters:
+  job: null
+  pool:
+    name: Hosted Ubuntu 1604
+  imageName: null
+
+jobs:
+- job: ${{ parameters.job }}
+  pool: ${{ parameters.pool }}
+  timeoutInMinutes: 30
+  variables:
+    # Use ":z" to set selinux flag for sharing in build-owned root dir. https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
+    docker.agentSrc.map: -v $(Build.SourcesDirectory):/agentSrc:z
+    docker.agentSrc.work: -w /agentSrc
+    docker.run: docker run --rm
+    docker.root.map: -v $(rootDirectory):/root:z
+    docker.src.map: -v $(rootDirectory)/source-build-reference-packages/src:/src:z
+    docker.src.work: -w /src
+    docker.staging.map: -v $(stagingDirectory):/staging:z
+    imageName: ${{ parameters.imageName }}
+    rootDirectory: $(Build.SourcesDirectory)/..
+    stagingDirectory: $(rootDirectory)/source-build-reference-packages/staging
+
+  steps:
+  - template: ../steps/docker-cleanup-linux.yml
+
+  # Create working directory and copy source into it.
+  - script: |
+      $(docker.run) $(docker.root.map) $(docker.agentSrc.map) $(docker.agentSrc.work) $(imageName) bash -c '
+        mkdir -p /root/source-build-reference-packages/staging
+        rm -rf /root/source-build-reference-packages/
```

Comment:
Isn't this deleting the directory we just created?  (and some other stuff)

Reply:
That is definitely an error!  Thanks for catching it!